{{cookiecutter.github_repository}}/provisioner/roles/nginx/defaults/main.yml

```diff
@@ -25,4 +25,10 @@ ssl_session_timeout: '1d'
 ssl_session_cache: 'shared:SSL:50m'
 ssl_trusted_certificate: '{{ ssl_cert_dir }}/{{ domain_name }}/chain.pem'
 access_log_file: '/var/log/nginx/{{ domain_name }}.access.log'
-error_log_file: '/var/log/nginx/{{ domain_name }}.error.log'{% endraw %}
+error_log_file: '/var/log/nginx/{{ domain_name }}.error.log'
+
+# htpasswsd
+# docs username, password
+htpasswd_file_path: '/etc/nginx/.htpasswd'
```

Comment:
Is it possible to create this file with `Environment` namespace to create separate files for `dev` and `qa` environment on same server?

Reply:
Yes. It should be. I will look into it.